src/popup/components/AddressTruncated.vue

```diff
@@ -70,11 +79,25 @@ export default defineComponent({
 
 <style lang="scss" scoped>
 @use '@/styles/typography';
+@use '@/styles/variables' as *;
 
 .address-truncated {
+  $this: &;
+
   display: flex;
   align-items: center;
   user-select: none;
+  color: inherit;
+
+  &.hoverable {
```

Comment:
![hoverEffect](https://github.com/user-attachments/assets/3f49e032-4fa1-4aff-9aea-8e15c4f2fe87)
Hover effect doesn't seems to work on the name itself.

Reply:
Fixed. Thank you for letting me out. Maybe you want to retest @CedrikNikita ?